src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -120,6 +120,8 @@ public SwitchParameter NoClobber
 
```

Comment:
@IISResetMe, your last commit had 1 failures in `PowerShell-CI-linux`
CliXml test.Import-CliXML.test follow-up for WinBlue: 161470 - Export-CliXml errors in WhatIf scenarios
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/clixml.tests.ps1: line 169
169:             Export-Clixml -Path $testPath -InputObject "string" -WhatIf

```